CHANGELOG.md

```diff
@@ -25,10 +25,14 @@ BUG FIXES:
   `vapp_template_lease` to define lease related parameters of depending entities - [GH-432]
 * `resource/vcd_vapp_vm` `network` block changes caused MAC address changes in existing NICs
   [GH-436,GH-407]
+* Fix a potential data race in client connection caching when VCD_CACHE is enabled [GH-453]
 
 NOTES:
+
 * Drop support for vCD 9.0
 * Bump terraform-plugin-sdk to v1.5.0 [GH-442]
+* `make seqtestacc` and `make test-binary` use `-race` flags for `go test` to check if there are no data races.
```

Comment:
The target name`seqtestacc` sends a bit different message - it looks like the test is a "downgrade" from the usual `testacc` as you don't see from the name that it uses `-race`. Maybe better:

- `testaccrace`
- `testaccraceseq`
- `testaccseqrace`
- `testacc-seq-race`
- `testacc-race-seq` (for me this one is clearest)

To me it looks like it and related functions should start with `test...` to be consistent:

![Screenshot 2020-02-17 at 09 28 38](https://user-images.githubusercontent.com/11330689/74632372-e5605b00-5167-11ea-9c93-824cc0597081.png)

Reply:
Ok, we can change it. The `make seqtestacc` has actually been here for a while now. Before none of these had `-race` flag, but now I added it to `make seqtestacc`. This may be a good opportunity to rename it after all then.

`testacc-race-seq` suits me.